care/emr/management/commands/load_fixtures.py

```
@@ -1,1886 +1,298 @@
-        if kwargs:
-            data.update(kwargs)
-
-        org_spec = OrganizationWriteSpec(**data)
-        org = org_spec.de_serialize()
-        org.created_by = super_user
-        org.updated_by = super_user
-        org.save()
-        return org
+    def load_fixtures(self, base):  # noqa: PLR0915, PLR0912
+        log = self.stdout.write
+        password = "Ohcn@123"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "password"

(S105)